create-register.sh

```diff
@@ -0,0 +1,257 @@
+#!/bin/sh
```

Comment:
should probably be `/bin/bash` -- I suspect you'll have used some bash-specific features